Sources/App/Controllers/Karma/KarmaController+Slack.swift

```diff
@@ -43,59 +44,55 @@ extension KarmaController {
                     }
                 }
 
-                let attachments = karma.map { KarmaMessage(user: $0.id ?? "", count: $0.count).karmaAttachment() }
                 return try req.client().post(responseUrl) { beforePost in
-                    let karmaResponse = KarmaResponse(attachments: attachments)
+                    let karmaResponse = KarmaStatusResponse(forSlashCommandWithKarmaStatuses: karma)
                     try beforePost.content.encode(json: karmaResponse)
                 }
-            }
-            .catch {
+            }.catch {
```

Comment:
I looked into the Vapor/NIO frameworks for guidance on this. Seems this way is the way to do it

Reply:
Confused ?